Application/src/visuals/Draw.java

```diff
@@ -22,6 +23,9 @@
     private ArrayList<Message> messages = new ArrayList<>(); // Stores the messages between nodes.
     private int offset; // Used for message ordering
     private int class_size = 0; // Used for message positioning
+	
+    //stores an animated gif file specifically made for this application, which contains an 8-bit animation of a sky/ocean view
+    private static Image animatedBackground = new Image("resources/SkyGIF.gif");
```

Comment:
Does this need to be static?

Reply:
in relation of memory and performance; it is static to be loaded only once in the canvas

Reply:
Congratulations